include/RAJA/policy/cuda/atomic.hpp

```diff
@@ -25,6 +25,10 @@
 #include <stdexcept>
 #include <type_traits>
 
+#if __CUDA__ARCH__ >= 600 && __CUDACC_VER_MAJOR__ >= 11 && __CUDACC_VER_MINOR__ >= 6
```

Comment:
Do we have test coverage for this case?

Reply:
We build with nvcc 11.7 and I thought our blueos platforms supported sm_70, so I think so.

Reply:
Can you add a define to enable things if this is true, its nicer than repeating the conditional.